src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import core.basesyntax.modul.Circle;
+import core.basesyntax.modul.IsoscelesTrapezoid;
+import core.basesyntax.modul.Rectangle;
+import core.basesyntax.modul.RightTriangle;
+import core.basesyntax.modul.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 25;
+    private Random rnd = new Random();
+
+    public Figure getRandomFigure() {
+        Figure[] figures = {new Square(rnd.nextInt(BOUND)),
+                new Circle(rnd.nextInt(BOUND)),
+                new RightTriangle(rnd.nextInt(BOUND), rnd.nextInt(BOUND)),
+                new IsoscelesTrapezoid(rnd.nextInt(BOUND),rnd.nextInt(BOUND),rnd.nextInt(BOUND)),
+                new Rectangle(rnd.nextInt(BOUND), rnd.nextInt(BOUND))};
+        return figures[rnd.nextInt(figures.length)];
```

Comment:
you create array with 5 figures just to pick one of them. Lets use switch-case construction. We can create 5 private methods getRandomCircle(), getRandomSquare(), ... and call them in switch-case construction. And we should set random color to figure at the time of its creation